Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -356,6 +358,22 @@ defineSuite([
         });
     });
 
+    it('loads tileset with extras', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            expect(tileset.extras).toBeDefined();
+            expect(tileset.root.extras).not.toBeDefined();
+
+            var taggedChildren = 0;
+            for(var i = 0;i < tileset.root.children.length; i++) {
+                if (defined(tileset.root.children[i].extras)) {
+                    taggedChildren ++;
```

Comment:
Similarly add the `++` before the variable here too to match the convention:

`++taggedChildren;`